### DISCUSSION TITLE
Custom Quantization export

### DISCUSSION THREAD

**Discussion by MohamedA95** [microsoft/onnxruntime#17140] [Other Q&A]
Hi everyone, 
I have a model that was quantized using a custom library. I would like to export it to Onnx using the Operator-oriented method. I could not find in the docs how to do it, could you provide any leads?
Thanks in advance.

**Comment by skottmckay** [comment]
I don't know what the 'operator-oriented' method is. 

https://github.com/onnx/tutorials#converting-to-onnx-format lists the options for converting to onnx from various platforms.

**Reply by MohamedA95** [reply]
Hi Skottmckay, Thanks for your respond
As per the [documentation](https://onnxruntime.ai/docs/performance/model-optimizations/quantization.html#onnx-quantization-representation-format), there are two ways to represent a quantized onnx model: operator- or tensor-oriented. Now I have a custom quantization flow and I want to use onnx to save the model with some extra parameters at each node.
